src/libraries/System.Private.Xml/src/System/Xml/Xsl/XmlQueryType.cs

```diff
@@ -148,7 +148,7 @@ public bool NeverSubtypeOf(XmlQueryType baseType)
                 return false;
 
             // None is subtype of every other type
-            if (Count == 0)
+            if (IsEmpty)
```

Comment:
@krwq does this property resembles the same as the previous expression?

Reply:
cc @buyaa-n

Reply:
it doesn't really look like tests the same thing, i would revert and suppress the warning